cookbooks/continental_compression_with_imposed_faults/continental_compression_with_imposed_faults.prm

```diff
@@ -0,0 +1,437 @@
+                         noninitial_plastic_strain, \
+                         viscous_strain, \
+                         crust_upper, \
+                         crust_lower, \
+                         mantle_lithosphere, \
+                         asthenosphere
+  set Types of fields = strain, \
+                        strain, \
+                        strain, \
+                        chemical composition, \
+                        chemical composition, \
+                        chemical composition, \
+                        chemical composition
+  set Compositional field methods = particles
+  set Mapped particle properties  = plastic_strain: plastic_strain, \
+                                    noninitial_plastic_strain: noninitial_plastic_strain, \
+                                    viscous_strain: viscous_strain, \
+                                    crust_upper: initial crust_upper, \
+                                    crust_lower: initial crust_lower, \
+                                    mantle_lithosphere: initial mantle_lithosphere, \
+                                    asthenosphere: initial asthenosphere
+end
+
+# Initial values of different compositional fields
+# The upper crust (20 km thick), lower crust (20 km thick)
+# and mantle (60 km thick) are continuous horizontal layers
+# of constant thickness. The non initial plastic strain is
+# set to 0, while the initial plastic and viscous strain is
+# randomized between 0.5 and 1.5.
+subsection Initial composition model
+  set List of model names = function, world builder
+  set List of model operators = add, add
+
+  subsection Function
+    set Variable names      = x,y
+    set Function expression = if(x>40.e3 && x<360.e3 && y>360.e3, 0.5 + rand_seed(1), 0); \
+                              0; \
+                              if(x>40.e3 && x<360.e3 && y>300.e3, 0.5 + rand_seed(1), 0); \
+                              if(y>=380.e3, 1, 0); \
+                              if(y<380.e3 && y>=360.e3, 1, 0); \
+                              if(y<360.e3 && y>=320.e3, 1, 0); \
+                              if(y<320.e3 && y>=0.e3, 1, 0);
+
+  end
+end
+
+# Composition: fixed on bottom (inflow boundary), free on sides and top
+subsection Boundary composition model
+  set Fixed composition boundary indicators = bottom, left, right
+  set List of model names = initial composition
+  set Allow fixed composition on outflow boundaries = true
+end
+
+# Temperature boundary conditions
+# Top and bottom (fixed) temperatures are consistent with the initial temperature field
+# Note that while temperatures are specified for the model sides, these values are
+# not used as the sides are not specified "Fixed temperature boundaries".  Rather,
+# these boundaries are insulating (zero net heat flux).
+subsection Boundary temperature model
+  set Fixed temperature boundary indicators = bottom, top
+  set List of model names = box
+  subsection Box
+    set Bottom temperature = 1669
```

Comment:
If it's worth explaining, can you mention why you chose this very specific value for the bottom temperature?